index.html

```diff
@@ -56,6 +59,13 @@
     },
     includePermalinks: false,
 
+    preProcess: [
+      prepare_reqlist
```

Comment:
```suggestion
      //prepare_reqlist
```

Reply:
Suggest we merge this rather than just leaving it floating out there as a PR, we can uncomment when Editors need to understand the reqlist.